frontend/src/components/PatientForm.js

```diff
@@ -54,7 +53,7 @@ const PatientForm = () => {
         <input type="text" onChange={(e) => setRasa(e.target.value)} value = {rasa}/>
 
         <label> Id właściciela pacjenta:</label>
-        <input type="number" onChange={(e) => setId_owner(e.target.value)} value = {id_owner}/>
+        <input type="number" onChange={(e) => setIdWlasciciela(e.target.value)} value = {id_wlasciciela}/>
```

Comment:
Z perspektywy czasu mogliśmy jednak po angielsku zrobić te nazwy w bazie, bo gryzie mi się taki zapis xD
Ale teraz to już nieistotne bo nie ma co refaktorować ponownie